Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.m

```diff
@@ -471,7 +487,13 @@ - (NSURLSessionDataTask *)completeCheckout:(BUYCheckout *)checkout withApplePayT
 	}
 	else {
 		NSString *tokenString = [[NSString alloc] initWithData:token.paymentData encoding:NSUTF8StringEncoding];
-		NSDictionary *paymentJson = @{ @"payment_token" : @{ @"payment_data" : tokenString, @"type" : @"apple_pay" }};
+		NSDictionary *paymentJson = @{
+									  @"payment" : @{
```

Comment:
Its more consistent with the credit card checkout if both boilerplate JSON creation is done in place, no?